views/js/runner/plugins/tools/highlighter/plugin.js

```diff
@@ -64,8 +68,21 @@ define([
             var testConfig      = testData.config || {};
             var pluginShortcuts = (testConfig.shortcuts || {})[this.getName()] || {};
             var hasHighlights   = false;
+            var logger          = loggerFactory('highlighterPlugin');
 
-            var highlighter = highlighterFactory();
+            /**
+             * @var {Object} collection - Highlighters collection
+             * See taoQtiTest/views/js/runner/plugins/tools/highlighter/collection.js
+             */
+            var collection = highlighterCollection();
```

Comment:
could you please rename `collection` by `highlighters` or `highlighterCollection` ?

Reply:
Done.